pvlib/pvsystem.py

```diff
@@ -148,7 +148,7 @@ class PVSystem:
          Describes the module's construction. Valid strings are 'glass_polymer'
          and 'glass_glass'. Used for cell and module temperature calculations.
 
-    module_parameters : None, dict or Series, default None
+    module_parameters : None, dict or Series, optional
         Module parameters as defined by the SAPM, CEC, or other.
 
     temperature_model_parameters : None, dict or Series, default None.
```

Comment:
```suggestion
    temperature_model_parameters : None, dict or Series, optional
```

Reply:
Thanks for spotting that None, I've regexed all the files to find and replace them.